packages/ibm-products/src/components/FilterPanel/FilterPanelCheckboxWithOverflow/FilterPanelCheckboxWithOverflow.test.js

```diff
@@ -0,0 +1,126 @@
+  render(
+    <FilterPanelCheckboxWithOverflow
+      data-testid={dataTestId}
+      id={id}
+      labelText={labelText}
+      {...{ ...rest }}
+    >
+      <OverflowMenuItem
+        itemText="Option 1"
+        // onClick={(event) => {
+        //   action('onClick Option 3')(event);
+        // }}
+      />
+    </FilterPanelCheckboxWithOverflow>
+  );
+
+describe(componentName, () => {
+  it('renders a component FilterPanelCheckboxWithOverflow', async () => {
+    renderComponent();
+
+    // Test wrapper div exists.
+    expect(screen.getByTestId(dataTestId)).toBeInTheDocument();
+    // Test checkbox exists.
+    expect(screen.getByRole('checkbox')).toBeInTheDocument();
+  });
+
+  it('renders a count', async () => {
+    const filterPanelLabelClass = `${pkg.prefix}--filter-panel-label`;
+    const { container } = renderComponent({ count: count });
+    expect(
+      container.querySelector(`.${filterPanelLabelClass}__count`).textContent
+    ).toBe('5');
+  });
+
+  it('shows overflow menu button on hover', async () => {
+    const user = userEvent.setup();
+    const { container } = renderComponent();
+
+    const label = screen.getByText(labelText);
+    await act(() => user.hover(label));
+
+    const menuButton = container.querySelector(
+      `.${blockClass}__overflow-button`
+    );
+    expect(menuButton).toBeInTheDocument();
+  });
+
+  it('shows overflow menu on click', async () => {
+    const user = userEvent.setup();
+    const { container } = renderComponent();
+
+    // 1. Hover over filter label to show menu button.
+    const label = screen.getByText(labelText);
+    await act(() => user.hover(label));
+
+    // 2. Click menu button to show menu.
+    const menuButton = container.querySelector(
+      `.${blockClass}__overflow-button`
+    );
+    await act(() => user.click(menuButton));
+
+    // 3. The Carbon's OverflowMenu is initially set as
+    //    `style="visibility:hidden"`, before being made
+    //    visible to the user.
+    const menu = screen.getByRole('menu', { hidden: true });
+    expect(menu).toBeInTheDocument();
+  });
+
+  it('has no accessibility violations', async () => {
+    const { container } = renderComponent();
+    expect(container).toBeAccessible(componentName);
+    expect(container).toHaveNoAxeViolations();
+  });
+
+  it('applies className to the containing node', async () => {
+    const { container } = renderComponent({ className: className });
+    expect(container.querySelector(`.${blockClass}`)).toHaveClass(className);
+  });
+
+  it('adds additional props to the containing node', async () => {
+    renderComponent({ 'data-testid': dataTestId });
+    screen.getByTestId(dataTestId);
+  });
+
+  it('forwards a ref to an appropriate node', async () => {
+    const ref = React.createRef();
+    renderComponent({ ref: ref });
+    expect(ref.current).toBeInTheDocument();
+  });
+
+  // it('adds the Devtools attribute to the containing node', async () => {
```

Comment:
Are you able to add this test back in? We try to include this for every component.